src/cascadia/TerminalApp/App.cpp

```diff
@@ -57,6 +57,37 @@ namespace winrt::TerminalApp::implementation
         // registered?" when it definitely is.
     }
 
+    void App::Initialize()
+    {
+        if (this->outer())
+        {
+            this->outer()->AddRef();
+        }
+
+        _windowsXamlManager = winrt::Windows::UI::Xaml::Hosting::WindowsXamlManager::InitializeForCurrentThread();

```

Comment:
Do we need to have the App do this? Did this need to move from it's original location in `main.cpp`?

Reply:
Please never call this if you are using XamlApplication, XamlApplication will do this for you.

There is a bug where if the order off calls into InitializeForCurrentThread/Close is not exactly right, we may end up not clearing the core or cleaning the core at the wrong time.